.github/workflows/proof_ci_resources/config.yaml

```diff
@@ -0,0 +1,7 @@
+cadical-tag: latest
+cbmc-version: latest
+cbmc-viewer-version: latest
+kissat-tag: latest
+litani-version: latest
+proofs-dir: verification/cbmc/proofs
+run-cbmc-proofs-command: ./run-cbmc-proofs.py
```

Comment:
starter-kit is not part of this list, so I assume we will continue updating it as we always do: submitting a PR after applying the script the update it. Is that correct?

Reply:
This file is considered part of the starter kit, it's in the same repo :) so whenever we update the starter kit, the relevant files will change anyway